extension/src/api/payment/payment.controller.js

```diff
@@ -30,12 +30,15 @@ async function processRequest(request, response) {
       paymentObject,
       authToken
     )
-
-    return httpUtils.sendResponse({
+    const result = {
       response,
       statusCode: paymentResult.success ? 200 : 400,
-      data: paymentResult.data,
-    })
+      data: paymentResult.success
```

Comment:
Do we need this flag if we can check for existence of errors instead?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/656/commits/3706831713c202a1f0bc0671a40ecc4ffe52a098